daft/dataframe/dataframe.py

```diff
@@ -1203,7 +1415,23 @@ def into_partitions(self, num: int) -> "DataFrame":
         (i.e. if there are 2 partitions, and change it into 3, this function will just split the bigger one)
 
         Example:
+            >>> import daft
+            >>> df = daft.from_pydict({"x": [1, 2, 3], "y": [4, 5, 6], "z": [7,8,9]})
             >>> df_with_5_partitions = df.into_partitions(5)
+            >>> df_with_5_partitions.show()
```

Comment:
Instead of doing a `.show()`, do a `.num_partitions()` so we can see the new number of partitions.